lib/config/default.js

```diff
@@ -52,6 +52,13 @@ defaultConfig.definition = () => ({
    * You shouldn't need to change this.
    */
   host: '',
+  /**
+   * Endpoint to send OpenTelemetry spans to.
+   *
+   * This should be automatically deduced from your region and other
+   * settings, but if desired, you can override it.
+   */
+  otlp_endpoint: '',
```

Comment:
i would defer documenting this in docs.newrelic.com for now until we get feedback on our approach for otel + nr agent

Reply:
Do you mean I should delete this docstring?